pkg/autodiscover/autodiscover_pv.go

```diff
@@ -31,3 +31,11 @@ func getPersistentVolumes(oc corev1client.CoreV1Interface) ([]corev1.PersistentV
 	}
 	return pvs.Items, nil
 }
+
+func getPersistentVolumeClaims(oc corev1client.CoreV1Interface) ([]corev1.PersistentVolumeClaim, error) {
+	pvcs, err := oc.PersistentVolumeClaims("").List(context.TODO(), metav1.ListOptions{})
```

Comment:
Suggestion: As pvcs are namespaced objects, shouldn't we List() the ones in the configured CNF's namespaces?

Reply:
I kept this namespace field empty because I figured this was our "general" PVC collecting function and we would filter later like we do with the other resources.